DocNet.Core/Markup/Views/assemblyViewTemplate.html

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<html>
+
+<head>
+<meta charset="utf-8">
+	<link href="../css/bootstrap.min.css" rel="stylesheet">
+	<link href="../css/bootstrap-theme.min.css" rel="stylesheet">
+	<link href="../css/custom.css" rel="stylesheet">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
We shouldn't be referencing external resources. Everything must be local so that things are still readable when the user has no internet access.